src/components/exchange/WarningSigarraSync.tsx

```diff
@@ -0,0 +1,40 @@
+
+import { useState } from "react";
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "../../components/ui/alert-dialog"
+type Props = {
+  no_changes: boolean
+}
+
+export const WarningSigarraSync = ({ no_changes }: Props) => {
+  const [open, setOpen] = useState<boolean>(true);
+  return (
+    <>
+      {no_changes ?
```

Comment:
In order for the code to be more consistent as well for the fact that javascript does not normally use snake case for variable names, please change this to `noChanges`.